make/harbor.yml.tmpl

```diff
@@ -86,6 +86,9 @@ trivy:
   # This option doesn’t affect DB download. You need to specify "skip-update" as well as "offline-scan" in an air-gapped environment.
   offline_scan: false
   #
+  # Comma-separated list of what security issues to detect. Possible values are `vuln`, `config` and `secret`. Defaults to `vuln`.
+  security_check: vuln
```

Comment:
If I want to enable `config` or `secret`, what should input? `vuln, secret`?

Reply:
correct. comma-separated without space:

Reply:
Thanks, maybe we can add comments for this.